segmenters/nlp/SpacySentencizer/tests/test_spacysentencizer.py

```diff
@@ -9,23 +9,21 @@ def multilingual_model_name():
     return "xx_sent_ud_sm"
 
 
-@pytest.fixture
-def multilingual_tests():
-    return [
+@pytest.mark.parametrize(
```

Comment:
You can separatr these in 2 parameters, one for the input and one for the expected value